shab.nix

```diff
@@ -0,0 +1,25 @@
+{ bash, stdenv, lib, ... }:
+
+stdenv.mkDerivation {
+  name = "shab";
+  version = "1";
+  src = ./.;
```

Comment:
`lib.filterSource ./.`, ideally it would exclude everything except ./shab and the ./test folder to minimize rebuilds. Because our massive monorepo takes too long to rebuild otherwise :p